README.md

```diff
@@ -22,17 +22,17 @@ To get a local copy up and running follow these simple example steps.
 
 ### Setup & Install
 
-Download sources from [Github](https://github.com/amentetteh/amen-tetteh-portfolio/main.zip) or throught the command: 
+Download sources from [Github](https://github.com/amentetteh/fintech-challenge-final-web-site/dev.zip) or throught the command: 
 ```
-git clone https://github.com/amentetteh/amen-tetteh-portfolio.git
+git clone https://github.com/amentetteh/fintech-challenge-final-web-site.git
 
 ```
 
 ### Usage
 
 After downloading source on your local machine, you can simply edit the HTML and CSS files from the folder. To preview the changes you make to the code, you can open the index.html file in your web browser.
 
-[Go Live] (https://amentetteh.github.io/)
+[Go Live](https://amentetteh.github.io/fintech-challenge-final-web-site)
```

Comment:
- [ ] Great job working on the requested changes but I think this link is not working so kindly fix it and make sure that it is working fine since it is just show me 404 not found like this

![image](https://user-images.githubusercontent.com/58553711/198856673-462fed63-f653-417b-9549-e7c5e3dfe9a6.png)